src/server/com/yetanalytics/datasim/server.clj

```diff
@@ -153,12 +159,15 @@
               (assoc context
                      :response
                      (try
-                       {:status  200
-                        :headers {"Content-Type"        "application/json; charset=utf-8"
-                                  "Content-Disposition" (format "attachment; filename=\"simulation.json\"")}
```

Comment:
was `"Content-Disposition"` removed intentionally?

Reply:
So i think this approach was abandoned in the actual implementation. If you look at the FE code it takes the Blob that comes over XHR and then marshalls it into a downloaded file complete with filename.

Reply:
:+1: